middleware.js

```diff
@@ -0,0 +1,28 @@
+import { createMiddlewareClient } from '@supabase/auth-helpers-nextjs';
```

Comment:
Is it possible to move this file into the `src` folder and also use rename to a typescript file using `.ts`?

Reply:
✅